src/pyspiffe/bundle/jwt_bundle/exceptions.py

```diff
@@ -4,12 +4,24 @@
 from pyspiffe.exceptions import PySpiffeError
 
 
-class AuthorityNotFoundError(PySpiffeError):
-    """Raised when an authority is not found associated with a key_id.
+class JwtBundleError(PySpiffeError):
+    """Top level exception for the X509Bundle module. """
 
-    Attributes:
-        message: Message describing the error.
-    """
+    def __init__(self, message: str) -> None:
+        """Creates an instance of X509BundleError.
+
+        Args:
+            message: Message describing the error.
+        """
+
+        self.message = message
+
+    def __str__(self) -> str:
+        return self.message
+
+
+class AuthorityNotFoundError(JwtBundleError):
+    """Raised when an authority is not found associated with a key_id."""
 
     _MESSAGE = 'Key ({}) not found in authorities.'
 
```

Comment:
nit: all code is using single quotes, we should use here too. by here, I mean the default parameter value for str; str = 'not specified'.